base_layer/core/src/blocks/genesis_block.rs

```diff
@@ -63,6 +64,99 @@ pub fn get_genesis_block(network: Network) -> ChainBlock {
     }
 }
 
+pub fn get_stagenet_genesis_block() -> ChainBlock {
+    let block = get_stagenet_genesis_block_raw();
+
+    let accumulated_data = BlockHeaderAccumulatedData {
+        hash: block.hash(),
+        total_kernel_offset: block.header.total_kernel_offset.clone(),
+        achieved_difficulty: 1.into(),
```

Comment:
Is this the actual difficulty of the block?

Reply:
It's the difficulty emse has so I left it as is.